addons/ui/ACE_UI.hpp

```diff
@@ -0,0 +1,177 @@
+// IDD, elements[]s, Element (must be string to compare to changed setting name), Selective Type (0-both, 1-ground, 2-vehicle)
+class ACE_UI {
+    class weaponName {
+        idd = 300;
+        elements[] = {118};
+        condition = QUOTE(ACE_player == vehicle ACE_player);
+        class conditions {};
```

Comment:
No, if one is false it will set the element to hidden, if they are all true it will show the element. The difference between the main condition and the conditions class is that when the control is shown by the engine you still need to make sure you hide/show it every time the UI gets updated by the engine. Meanwhile the main condition will exit right away if the control is not getting rendered at all.